### PR TITLE
Closes #1 - Create Index.html

### LINKED ISSUE #1: Create Index.html
blAVLA

### PR DESCRIPTION
**Pull request by aaandrianaaa**

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+</head>
+<body> </body>
```
Comment (BohdanZahirnyi) [inline Index.html]: figovo

**Review by BohdanZahirnyi — CHANGES_REQUESTED** [review]

**Review by oleksiistetsiuk — CHANGES_REQUESTED** [review]
rewrite body tag

**Review by mod1fy — APPROVED** [review]
All ok!

**Comment by bohdansydor** [comment]
***

File: Index.html (hunk 1 above)
Comment (Dreylak) [reply Index.html]: sdfdf

**Review by Dreylak — CHANGES_REQUESTED** [review]

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```
Comment (drudevych) [inline Index.html:1]: Ya tut

**Review by drudevych — CHANGES_REQUESTED** [review]

**Review by artur-britvin — CHANGES_REQUESTED** [review]
pofix xe bigom

**Review by BohdanZahirnyi — APPROVED** [review]

**Review by Dreylak — APPROVED** [review]

**Review by oleksiistetsiuk — APPROVED** [review]

**Review by drudevych — APPROVED** [review]

**Review by bohdansydor — APPROVED** [review]

**Review by artur-britvin — APPROVED** [review]

**Pull request merged**